simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/IllegalAttributeTypeException.java

```diff
@@ -0,0 +1,9 @@
+package com.evbox.everon.ocpp.simulator.station.component.exception;
+
+public class IllegalAttributeTypeException extends RuntimeException {
+
+    public IllegalAttributeTypeException(String value) {
+        super("Unknown attribute type: " + value);
```

Comment:
Please dont put additional logic to constructors

Reply:
ok, will fix